instance/openstack_utils.py

```diff
@@ -39,7 +39,8 @@
 
 # Data objects ################################################################
 
-FailedContainer = namedtuple('FailedContainer', ['name', 'number_of_failures'])
+FailedContainer = namedtuple(
+    'FailedContainer', ['name', 'number_of_failures', 'extra_information'])
```

Comment:
Looks like this change broke some of the tests in instance/tests/test_openstack_utils.py.
Might this affect other uses of FailedContainer? If not we can change those tests too.

Reply:
@kaizoku Thanks for pointing that out!

`FailedContainer` is used only in `openstack_utils.download_swift_account` function which itself is used only in `backup_swift.tasks.do_backup_swift` so I've just updated the failing tests.